### PR TITLE
Add Clear Linux OS setup instructions.

### PR DESCRIPTION
**Pull request by mrkz**
This PR adds the setup for [Clear Linux OS](https://clearlinux.org/)

Signed-off-by: Simental Magana, Marcos <marcos.simental.magana@intel.com>

### CODE REVIEW

File: data/distro.yml [hunk 1]
```diff
@@ -342,3 +342,15 @@
         <p>Note: graphical installation of Flatpak apps may not be possible with Raspbian.</p>
       </li>
     </ol>
+
+- name: "Clear Linux"
+  logo: "clearlinux.svg"
+  info: >
+    <ol class="distrotut">
+      <!-- Flatpak is installed by default when installing the desktop bundle. -->
+      <p>Flatpak is installed and flathub repository is pre-configured by default on Clear Linux when installing the desktop bundle.</p>
```
[the remark this answers is not in the document]
Comment (mrkz) [reply data/distro.yml]: Changed, thanks for the review!

**Comment by AsciiWolf** [comment]
Any news?

**Comment by mrkz** [comment]
I pushed the fixes already, I thought a notification would be sent,
probably was not the case.

The requests were addressed at any case. Thanks for your time!

**Comment by alexlarsson** [comment]
Looks good to me, i've merged it. However, we need to rebuild and redeploy the site, which will happen next week when @jimmac is back.

**Comment by jimmac** [comment]
Deployed to github.

**Comment by alexlarsson** [comment]
Should be live now

**Pull request merged**